commons/ava_networks/high_level_gecko_client.go

```diff
@@ -25,17 +25,20 @@ const (
 )
 
 type HighLevelGeckoClient struct {
-	client    *gecko_client.GeckoClient
-	geckoUser *GeckoUser
+	client                   *gecko_client.GeckoClient
+	geckoUser                *GeckoUser
+	networkAcceptanceTimeout time.Duration
```

Comment:
given that we're going to go with one timeout across the board, can we add a comment explaining why this makes sense (since future me is def going to be confused too haha) - i.e. because we expect all things that operate on the network to take the same amount of time